assets/src/components/ladder.tsx

```diff
@@ -201,29 +188,43 @@ const yForVehicle = (
   return null
 }
 
-const Triangle = ({
-  x,
-  y,
-  vehicleDirection,
-  isSelected,
-}: {
-  x: number
-  y: number
-  vehicleDirection: VehicleDirection
-  isSelected: boolean
-}) => {
-  const points =
-    vehicleDirection === VehicleDirection.Up
-      ? [[x, y - 10], [x - 15, y + 10], [x + 15, y + 10]]
-      : [[x, y + 10], [x + 15, y - 10], [x - 15, y - 10]]
-  const pointsString = points.map(xy => xy.join(",")).join(" ")
+const Triangle = ({ isSelected }: { isSelected: boolean }) => {
   const selectedClass = isSelected ? "selected" : ""
   return (
-    <polygon
-      className={`m-ladder__vehicle-triangle ${selectedClass}`}
-      points={pointsString}
-    />
+    <g className="m-ladder__vehicle-icon" transform={`scale(0.625)`}>
+      <path
+        className={`m-ladder__vehicle-triangle ${selectedClass}`}
+        d="m27.34 9.46 16.84 24.54a4.06 4.06 0 0 1 -1 5.64 4.11 4.11 0 0 1 -2.3.71h-33.72a4.06 4.06 0 0 1 -4.06-4.11 4 4 0 0 1 .72-2.24l16.84-24.54a4.05 4.05 0 0 1 5.64-1.05 4 4 0 0 1 1.04 1.05z"
+      />
+    </g>
```

Comment:
We'll eventually need to reuse this triangle in at least two other places: the vehicle properties panel and the pending box. This isn't a question we need to solve right now, but what's our plan for that?

Reply:
It is trivial to pull this out into a shared component.